webapp/src/DOMJudgeBundle/Command/ImportEventFeedCommand.php

```diff
@@ -931,7 +931,7 @@ protected function importTeam(array $event)
         // Also check if this is a private contest. If so, we need to add the team to the contest
         /** @var Contest $contest */
         $contest = $this->em->getRepository(Contest::class)->find($this->contestId);
-        if (!$contest->getPublic()) {
+        if (!$contest->getAllTeams()) {
```

Comment:
I think this is confusing: the name `getAllTeams` suggests that the method returns all teams in the contest, not whether all teams are part of the contest.

Reply:
Way better